test/array.specs.ts

```diff
@@ -0,0 +1,81 @@
+import "mocha";
+import fc from "fast-check";
+import { chunk } from "../src/array"; 
+
+describe("chunk: ", () => {
+    it("Should split arrays into chunks of the specified size", () =>
+      fc.assert(
+        fc.property(
+          fc.array(fc.integer(), { minLength: 1 }), //tableau non vide
+          fc.nat(10), // la taille de chunk entre 0 et 10
```

Comment:
Why do you need to specify that ? It should not be necessary

Reply:
Sorry I thought fc.nat need at least one argument :/